04/app.js

```diff
@@ -2,7 +2,54 @@ console.log('DOM');
 
 // struktura do wykorzystania w pętli
 const menuItems = [
-    {text: 'start', url: '/'},
-    {text: 'galeria', url: '/gallery'},
-    {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+  { text: 'start', url: '/' },
+  { text: 'galeria', url: '/gallery' },
+  { text: 'kontakt', url: '/contact' },
+];
+
+// z użyciem pętli
+const navElement = document.querySelector('nav');
+const ulElement = document.createElement('ul');
+
+menuItems.forEach((item) => {
```

Comment:
Jak widać zdecydowanie wygodniej i lepiej używać pętli :)
Czasami trzeba rozpisać sobie krok po kroku, aby użyć pętli - to nie jest źle, ale ostateczna wersja powinna być z wykorzystaniem pętli.